src/main/fc/fc_msp.c

```diff
@@ -1922,6 +1931,19 @@ static mspResult_e mspFcProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
         }
         break;
 
+    case MSP_TX_INFO:
+        
+
+    case MSP_SET_TX_INFO:
+        txInfo.year = sbufReadU8(src) + 2000;
```

Comment:
Why not just send the year as a 16-bit quantity? Avoids the need for the 2000 offset and `txInfo` is not tying to optimise for the structure size.

Reply:
I did, but it didn't work for some reason. Could be that the current msp.lua can not handle numbers higher than one byte.